### PR TITLE
moved back the v1 APIs to apis/

### PR DESCRIPTION
**Pull request by bitscuit**
because it did not matter where they were since operator-sdk would always generate the CRD YAML regardless.

Signed-off-by: Henry Li <henry.h.li@ibm.com>

### CODE REVIEW

**Comment by bitscuit** [comment]
No functional change. Testing would just be making sure cert-manager-operator still behaves as expected.

File: config/manifests/bases/ibm-cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -50,6 +60,15 @@ spec:
...
       version: v1alpha1
+    - description: 'An Issuer represents a certificate issuing authority which can
+        be referenced as part of `issuerRef` fields. It is scoped to a single namespace
+        and can therefore only be referenced by resources within the same namespace.
+        Documentation For additional details regarding install parameters check: https://ibm.biz/icpfs39install.
+        License By installing this product you accept the license terms https://ibm.biz/icpfs39license.'
+      displayName: Issuer
+      kind: Issuer
+      name: issuers.cert-manager.io
+      version: v1
```
Comment (qpdpQ) [inline config/manifests/bases/ibm-cert-manager-operator.clusterserviceversion.yaml:71]: same for this one?

Comment (bitscuit) [reply config/manifests/bases/ibm-cert-manager-operator.clusterserviceversion.yaml]: correct, removed

**Pull request merged**